KomodoPlatform/atomicDEX-Desktop
# 0.5.7.2 Release Candidate

Please perform the tests below as soon as possible, we aim to release this week to be ready for the airdrop on Friday at midnight.
As this is a minor release, we'll reserve full testplan suite for the 0.5.8 release. 
For 0.5.7.2, please focus on testing the following (along with basic smoke testing on all OS)

Changelog:
- Update version to 0.5.7.2
- Misc CI/CD updates
- Updated theme selectors for more extendable theme creation on whitelabels
- Improved text / background contrast in lite/dark theme modes
- Added a standardised `CancelButton` component
- Improved display of combo selectors in dex pro view
- Misc other theme tweaks
- Replaced Nomics chats with LiveCoinWatch charts
- Used more intuitive red/green buttons for buy/sell in pro view
- Used more intuitive wording for pro view order form field labels.
- Reduce log spam (info -> debug)
- Fixed bug introduced by `-segwit` coins order uuid duplication in orderbook/bestorders response
- Misc other minor bug fixes


To Test:
- Confirm correct app version is on display in settings (0.0.7.2)
- Confirm launch, login and swap on all OS (installer and portable, release archives only)
- Check all views for any examples of poor text / background contrast in both lite/dark theme. Screenshot and create an issue for anything found (this is not a blocker so may be resolved in 0.5.8)
- Check pro view chart display is sane at different window sizes with a variety of pairs. Unsupported pairs shoud show a text warning instead of the chart.
- Confirm "smart chain" icon is present in addressbook
- Confirm notifications button changes if no notifications listed
- Report any crashing! During dev, it seemed more stable since fixing the `-segwit` uuid duplication bug.

Anything found during testing which is not P0 / P1, please create an issue ticket so it can be fixed in 0.5.8 and not block or delay this release.
